File: scala-example/spring-boot-example/src/main/scala/com/github/poad/scala/springboot/example/config/JerseyConfig.scala
package com.github.poad.scala.springboot.example.config

import com.github.poad.scala.springboot.example.mapper.RawExceptionMapper
import org.glassfish.jersey.server.ResourceConfig
import org.springframework.stereotype.Component

@Component
class JerseyConfig extends ResourceConfig {
  packages(true, "com.github.poad.scala.springboot.example.resource")
  register(new RawExceptionMapper(), 1)
}

File: java-maven-example/spring-boot-data-doma2-jmx-example/src/main/java/com/github/poad/examples/java/webapps/JerseyConfig.java
package com.github.poad.examples.java.webapps;

import com.github.poad.examples.java.webapps.mapper.RawExceptionMapper;
import org.glassfish.jersey.server.ResourceConfig;
import org.springframework.stereotype.Component;

@Component
public class JerseyConfig extends ResourceConfig {

    public JerseyConfig() {
        packages(String.join(".", new String[]{
                        this.getClass().getPackage().getName(),
                        "resource", "impl"
                }),
                String.join(".", new String[]{
                        this.getClass().getPackage().getName(),
                        "resource"
                }));
        register(new RawExceptionMapper(), 1);
    }
}
